lib/helpers.js

```diff
@@ -49,7 +49,7 @@ function spawnWorker() {
 }
 
 function showError(givenMessage) {
-  var givenDetail = arguments.length > 1 && arguments[1] !== undefined ? arguments[1] : null;
+  var givenDetail = arguments.length <= 1 || arguments[1] === undefined ? null : arguments[1];
```

Comment:
I wish `babel` would make up its mind here... it keeps swapping the logic around.